CONTRIBUTING.md

```diff
@@ -83,4 +83,50 @@ libzmq, and inherits LGPL
 * perf examples are (c) iMatix, and LGPL
 
 [tornado]: http://www.tornadoweb.org
-[paramiko]: http://www.lag.net/paramiko
\ No newline at end of file
+[paramiko]: http://www.lag.net/paramiko
+
+## Setting up development environment
+
+To start development on pyzmq, you need a working development environment. Due to the dependancies on libzmq,
+this is a bit more involved than simply checking out the code.
```

Comment:
In almost any context, `pip install -r test-requirements -e .` will produce a development build. Specifying a non-bundled zeromq with `--zmq=...` is also possible, but not generally necessary.

Reply:
It looks like these instructions were taken mostly from the extremely out-of-date wiki. I updated the page with more accurate and simple instructions. This isn't nearly so complicated now.

Reply:
Brilliant! I tried your instructions on the wiki and they work perfectly, with just one small error:

`pip install -r test-requirements`
should be
`pip install -r test-requirements.txt`

Based on that, I'd say everything I added in CONTRIBUTING.md isn't necessary, and if you agree I'll just revert those changes. Or maybe link to the wiki? LMK.

Reply:
Yeah, feel free to revert these for now.

Reply:
I reverted changes to CONTRIBUTING so this doesn't apply now.